pom.xml

```diff
@@ -83,7 +83,7 @@
         <version.org.wildfly.checkstyle-config>1.0.7.Final</version.org.wildfly.checkstyle-config>
         <version.org.wildfly.common.wildfly-common>1.5.1.Final</version.org.wildfly.common.wildfly-common>
         <version.org.wildfly.core>9.0.1.Final</version.org.wildfly.core>
-        <version.org.wildfly.dist>14.0.1.Final</version.org.wildfly.dist>
+        <version.org.wildfly.dist>24.0.1.Final</version.org.wildfly.dist>
```

Comment:
@jamezp  I'm not sure why you need to bump WildFly version but should the core version be updated too?

If that's the case, WildFly 24.0.1.Final incorporated WildFly Core 16.0.1.Final.

Reply:
Upgraded to 17.0.1.Final https://github.com/wildfly/wildfly-maven-plugin/pull/201. The main reason for the upgrade from 14 is testing with Java 17 so I went with the latest release :)